wandb/apis/public/reports.py

```diff
@@ -106,16 +119,23 @@ def __repr__(self):
 
 
 class BetaReport(Attrs):
-    """BetaReport is a class associated with reports created in wandb.
+    """Represents a report object in the W&B system.
 
-    WARNING: this API will likely change in a future release
+    Provides access to report attributes (name, description, user, spec,
+    timestamps) and methods for retrieving associated runs,
+    sections, and for rendering the report as HTML.
 
     Attributes:
-        name (string): report name
-        description (string): report description;
-        user (User): the user that created the report
-        spec (dict): the spec off the report;
-        updated_at (string): timestamp of last update
+        id (string): Unique identifier of the report.
+        name (string): The name of the report.
```

Comment:
I'm thinking we shouldn't include this... the name the users are familiar with is the `display_name`

Reply:
What does an example `name` and `display_name` look like? I'm guessing the former is the report URL and the latter is the just the path name?

Reply:
Here is a good example:
```
  Entity: wandb
  Project: large_runs_demo
  ID: VmlldzoxMjU2OTExNA==
  Display Name: Test Report
  Name: 87ed6km4zhs
  Description: 
  URL: https://wandb.ai/wandb/large_runs_demo/reports/Test-Report--VmlldzoxMjU2OTExNA
  Created At: 2025-05-01T16:30:27Z
  Updated At: 2025-05-01T16:31:04Z
  Created by: peyton-sarmiento (peyton.sarmiento@wandb.com)
  ```